packages/react-examples/src/react-components/Migrations/utils.test.tsx

```diff
@@ -34,26 +35,48 @@ test('renders Markdown source blocks', () => {
   renderer.render(
     <CodeExample>
       {mockMDXSourceCodeBlock(`\`\`\`js
-                console.log("test");
-                \`\`\``)}
+        function test() {
+          console.log("test");
+        }
+      \`\`\``)}
     </CodeExample>,
   );
   const result = renderer.getRenderOutput();
 
   expect(result.props).toEqual({
-    // eslint-disable-next-line react/jsx-key
-    children: [<h3>JavaScript</h3>, <Source code={`console.log("test");`} language="js" />],
+    children: [
+      <h3>JavaScript</h3>,
+      <Source
+        code={`function test() {
+          console.log("test");
+        }`}
+        language="js"
+      />,
+    ],
   });
 });
 
 test('uses JSX for no header JSX source code blocks', () => {
   const renderer = createRenderer();
-  renderer.render(<CodeExample>{mockMDXSourceCodeBlock(`<Test title={"Example"} />`)}</CodeExample>);
+  renderer.render(
+    <CodeExample>
+      {mockMDXSourceCodeBlock(`<div>
```

Comment:
q: why this change (nested tree/div) ?

Reply:
Same as my previous response, wasn't testing multiline scenarios.